.golangci.yml

```diff
@@ -127,6 +127,9 @@ linters-settings:
     explicit-exhaustive-switch: true
     ignore-enum-members: "pmetric.MetricTypeEmpty"
 
+  predeclared:
+    ignore: copy
```

Comment:
I read the PR description, but I don't really understand it.

1. Please also explain here why.
2. Can we get rid of it in the future? if yes, file an issue explain what we need to do to remove this exception and add a link here.